dmriprep/workflows/dwi/base.py

```diff
@@ -201,6 +203,43 @@ def _bold_reg_suffix(fallback):
         ])
         # fmt: on
 
+    # Eddy distortion correction
+    eddy_wf = init_eddy_wf(debug=config.execution.debug)
+    eddy_wf.inputs.inputnode.metadata = layout.get_metadata(str(dwi_file))
+
+    ds_report_eddy = pe.Node(
+        DerivativesDataSink(
+            base_directory=str(config.execution.output_dir),
+            desc="eddy_corrected",
```

Comment:
```suggestion
            desc="eddy",
```
desc should not have underscores (or any entity FWIW)

Reply:
Thanks, fixed in b0024d5e554234f0bdf1144a89846e92752568a3.